app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def post_count
+    posts.count
+  end
```

Comment:
- [Required] I think it would be better to make this method work as specified in the project requirements, the `User` model should have a method that returns `3` most recent posts. Your implementation would return the total number of a user's posts and not the `3` most recent ones, kindly rectify this.

## Project requirements
![Screenshot from 2023-04-07 02-38-55](https://user-images.githubusercontent.com/16392046/230512820-b3d59ec5-130b-48c9-abaa-561b2b03a38d.png)